fsharp-backend/src/LibExecutionStdLib/LibDate.fs

```diff
@@ -577,4 +577,19 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = SqlFunctionWithPrefixArgs("date_trunc", [ "'day'" ])
       previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "Date" "diff" 0
+      parameters = [ Param.make "endDate" TDate ""; Param.make "startDate" TDate "" ]
```

Comment:
maybe these should just be `a` and `b`? I think there's some usefulness to having `end` and `start` in there somewhere, since we're not taking the absolute value of the result. Thoughts?

Reply:
There's a sign in the output I think? If so, then end/start makes sense as it does matter which is which. a/b is for situations where there's no semantic meaning to the arguments.

Reply:
good point thanks